trax/models/download_model.py

```diff
@@ -0,0 +1,83 @@
+# coding=utf-8
```

Comment:
Please revert this file - we do not plan on maintaining file download APIs, it'll need to be done differently (or we can just do it later and leave it to be manual for now).

Reply:
done